packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWVoteWeightModule/CWVoteWeightModule.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { isMobile } from 'react-device-detect';
+import { CWText } from '../../cw_text';
+import { ComponentType } from '../../types';
+import CWIconButton from '../CWIconButton';
+import CWPopover, { usePopover } from '../CWPopover';
+import { CWButton } from '../cw_button';
+import './CWVoteWeightModule.scss';
+
+type VoteWeightModuleProps = {
+  voteWeight: number;
+  stakeNumber: number;
+  stakeValue: number;
+  denomination: string;
+};
+
+export const CWVoteWeightModule = ({
+  voteWeight,
+  stakeNumber,
+  stakeValue,
+  denomination,
```

Comment:
I am not sure about "currency" here but if this always be "ETH", we can make "ETH" as default here